models/public/pelee_coco/pelee_coco.md

```diff
@@ -0,0 +1,82 @@
+# Pelee
```

Comment:
Please, 
1. Use hyphen `-` instead of underscores `_`;
2. Use full model name as main header.

Reply:
updated, renamed